jwst/assign_wcs/miri.py

```diff
@@ -379,32 +377,33 @@ def alpha_beta2XanYan(input_model, reference_files):
 
     dict_mapper = {}
     sel = {}
+    # Since there are two channels in each reference file we need to loop over them
     for c in channels:
         ch = int(c[0])
         dict_mapper[tuple(wr[c])] = models.Mapping((2,), name="mapping_lam") | \
                    models.Const1D(ch, name="channel #")
-        map1 = models.Mapping((1, 0, 1, 0), name='map2poly')
-        map1._outputs = ('alpha', 'beta', 'alpha', 'beta')
-        map1._inputs = ('alpha', 'beta')
-        map1.inverse = models.Mapping((0, 1))
         ident1 = models.Identity(1, name='identity_lam')
         ident1._inputs = ('lam',)
         chan_v23 = v23[c]
         v23chan_backward = chan_v23.inverse
         del chan_v23.inverse
-        v23_spatial = map1 | chan_v23
-        v23_spatial.inverse = map1 | v23chan_backward
+        # This is the spatial part of the transform; tack on additional conversion to degrees
+        # Remove this degrees conversion once pipeline can handle v2,v3 in arcsec
+        v23_spatial=chan_v23 | models.Scale(1 / 3600) & models.Scale(1 / 3600)
+        v23_spatial.inverse = models.Scale(3600) & models.Scale(3600) | v23chan_backward
```

Comment:
OK, this is what I was worried abut but I see what you did here. You moved the Mapping to the reference file.
So no problem.